backend/packages/Upgrade/src/env.ts

```diff
@@ -83,6 +83,7 @@ export const env = {
     clientId: getOsEnvArray('GOOGLE_CLIENT_ID'),
     serviceAccountId: getOsEnvArray('GOOGLE_SERVICE_ACCOUNT_ID'),
     domainName: getOsEnvOptional('DOMAIN_NAME'),
+    authTokenRequired: toBool(getOsEnvOptional('GOOGLE_AUTH_TOKEN_REQUIRED')),
```

Comment:
Should this default to `true`? When we deploy this change, the variable won't be defined in the environments.

Reply:
I added a commit to update `app-infrastructure.yml` so that `GOOGLE_AUTH_TOKEN_REQUIRED` will be set to `true` when deployed.

Reply:
We're not using `projectBuilder.jenkinsfile` for anything anymore are we?

Reply:
Do we no longer use the file? Where do we get the SSM parameters from then? Is it not part of our repository?

Reply:
@danoswaltCL Please check the PR and let me know what you think.